web/config/initializers/shopify_app.rb

```diff
@@ -1,10 +1,34 @@
 # frozen_string_literal: true
 
 ShopifyApp.configure do |config|
+  config.webhooks = [
+    # After a store owner uninstalls your app, Shopify invokes the APP_UNINSTALLED webhook
+    # to let your app know.
+    { topic: "app/uninstalled", address: "api/webhooks/app_uninstalled" },
+
+    # NOTE 1: To register the URLs for the three GDPR topics that follow, please set the appropriate
+    # webhook endpoint in the 'GDPR mandatory webhooks' section of 'App setup' in the Partners Dashboard.
+    #
+    # NOTE 2: The code that processes these webhooks is located in the `app/jobs` directory.
+    #
+    # 48 hours after a store owner uninstalls your app, Shopify invokes this SHOP_REDACT webhook.
+    # https://shopify.dev/apps/webhooks/configuration/mandatory-webhooks#shop-redact
+    { topic: "shop/redact", address: "api/webhooks/shop_redact" },
+
+    # Store owners can request that data is deleted on behalf of a customer. When this happens,
+    # Shopify invokes this CUSTOMERS_REDACT webhook to let your app know.
+    # https://shopify.dev/apps/webhooks/configuration/mandatory-webhooks#customers-redact
+    { topic: "customers/redact", address: "api/webhooks/customers_redact" },
+
+    # Customers can request their data from a store owner. When this happens, Shopify invokes
+    # this CUSTOMERS_DATA_REQUEST webhook to let your app know.
+    # https://shopify.dev/apps/webhooks/configuration/mandatory-webhooks#customers-data_request
+    { topic: "customers/data_request", address: "api/webhooks/customers_data_request" },
+  ]
   config.application_name = "My Shopify App"
   config.old_secret = ""
-  config.scope = "read_products" # Consult this page for more scope options:
-  # https://help.shopify.com/en/api/getting-started/authentication/oauth/scopes
+  config.scope = "write_products,read_customers" # Consult this page for more scope options:
```

Comment:
This will conflict with a change I made to pull this from the toml file, sorry 😄